src/server/game/Entities/Player/Player.cpp

```diff
@@ -7687,17 +7697,14 @@ void Player::UpdateArea(uint32 newArea)
     else
         RemoveByteFlag(UNIT_FIELD_BYTES_2, 1, UNIT_BYTE2_FLAG_SANCTUARY);
 
-    if (isInn)
+    uint32 const areaRestFlag = (GetTeamId() == TEAM_ALLIANCE) ? AREA_FLAG_REST_ZONE_ALLIANCE : AREA_FLAG_REST_ZONE_HORDE;
+    if (area && areaFlags & areaRestFlag)
     {
         SetRestState(0);
-        if (sWorld->IsFFAPvPRealm())
```

Comment:
@Kitzunu are you sure we no longer need to check if `sWorld->IsFFAPvPRealm()` ?

I feel like this PR has been merged too fast :D

Reply:
What I dont understand is why someone is PMing every admin about it instead of making an issue.

However this is fixed tomorrow, I just havent published the branch